### DISCUSSION TITLE
Hide "new terminal" button

### DISCUSSION THREAD

**Discussion by drty2k** [Eugeny/tabby#8163] [Q&A] [answered]
How may hide "+" button and leave only "Profile&Connection" button? When i want open new connection often by mistake press + button and get not needed cmd tab - Its very annoiyng :(

**Comment by Eugeny** [comment, marked as answer]
If you only ever use Tabby as an SSH client, you can disable the "local" plugin in Settings -> Plugins which will disable local shells and the "plus" button too

**Reply by drty2k** [reply]
thanks)
